### ISSUE TITLE
System.FormatException when using WmtsParser

### ISSUE THREAD

**Issue by redcurry** [BruTile/BruTile#82]
I changed the WmtsSample in Mapsui to parse the following source:

https://api.nasa.gov/mars-wmts/catalog/Mars_Viking_MDIM21_ClrMosaic_global_232m/1.0.0/WMTSCapabilities.xml

But the Parse method in WmtsParser (BruTile library) throws the exception System.FormatException. Here's the topmost line in the stack trace:

```
System.FormatException: Input string was not in a correct format.
  at System.Number.StringToNumber (System.String str, System.Globalization.NumberStyles options, System.Number+NumberBuffer& number, System.Globalization.NumberFormatInfo info, System.Boolean parseDecimal) [0x0004d] in /Library/Frameworks/Xamarin.iOS.framework/Versions/11.6.1.3/src/mono/mcs/class/referencesource/mscorlib/system/number.cs:1074
```

**Comment by pauldendulk** [comment]
Thanks for the report.

I can reproduce it in a unit test. The problem seems to be that the MatrixWidth and MatrixHeight are specified as a double  while it should be an integer. So, there is an error in the capabilities. This is from the WMTS spec:

![image](https://user-images.githubusercontent.com/963462/35096322-100f1f76-fc4c-11e7-86a5-789ff9bcd742.png)

Which can be downloaded here:
http://www.opengeospatial.org/standards/wmts

BruTile should be robust enough for this though, so I will look for a solution.

**Comment by redcurry** [comment]
Thanks for looking into it.

**Comment by pauldendulk** [comment]
Published a 1.0.2 where this is fixed:
https://www.nuget.org/packages/BruTile/1.0.2
